docs/authorization-configuration.md

```diff
@@ -87,11 +107,11 @@ auth: {
 - **expirationCheckInterval** the number of seconds to pass between each check if the token is about to expire. The default value is `5` seconds.
 
 
-### Custom Authentication Provider
+## Custom authorization provider
 
-If you are using any authentication provider you can also implement the following functions for Luigi.
+If you are using another authorization provider, you can also implement the functions below for Luigi.
```

Comment:
```suggestion
If you use a custom authorization provider, you can also implement these functions for Luigi.
```
`You can` or `you must`? If it's optional, I think a one-sentence explanation here could be useful.